src/firebolt_cli/completer.py

```
@@ -73,6 +83,22 @@ def populate_table_and_column_names(self, cursor: Cursor) -> None:
         except (FireboltError, HTTPStatusError):
             pass
 
+    def populate_set_statemets(self, cursor: Cursor) -> None:
+        """
+        fetch all available set statements parameters
+        """
+        try:
+            cursor.execute("set use_standard_sql = 0")
+            cursor.execute("SELECT name FROM system.settings")
+            data = cursor.fetchall()
+
+            self.set_statements.extend(
+                (set_statement[0], "SET KEYWORD") for set_statement in data
+            )
+
+        except (FireboltError, HTTPStatusError):
+            pass
```

Let's add a DEBUG level output here so we're not just hiding an error.